grayskull/strategy/py_build.py

```diff
@@ -0,0 +1,158 @@
+"""
+Use pypa/build to get project metadata from a checkout. Create a recipe suitable
+for inlinining into the first-party project source tree.
+"""
+
+import logging
+import tempfile
+from importlib.metadata import PathDistribution
+from pathlib import Path
+
+import build
+from conda.exceptions import InvalidMatchSpec
+from conda.models.match_spec import MatchSpec
```

Comment:
Having to import conda will make grayskull more difficult to install.
Is it worth adding conda as dependency just for that function?

I haven't checked how requirements are parsed in the current code.